sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseLineListingSteps.java

```diff
@@ -85,7 +86,7 @@ public CaseLineListingSteps(
           webDriverHelpers.fillInWebElement(
               PERSON_ID_NAME_CONTACT_INFORMATION_LIKE_INPUT, caseName);
           webDriverHelpers.clickOnWebElementBySelector(CASE_APPLY_FILTERS_BUTTON);
-          webDriverHelpers.waitUntilNumberOfElementsIsReduceToGiven(CASE_DETAILED_TABLE_ROWS, 2);
+          TimeUnit.SECONDS.sleep(2); // wait for filter
```

Comment:
isn't the next method covering the wait until filter is displayed?

Reply:
Sleep works better in this case